_schema.graphql

```diff
@@ -7903,6 +7903,7 @@ type GeneItem implements Node {
 type GravityMutationError {
   detail: String
   error: String
+  fieldErrors: UpdateMyProfileMutationFieldError
```

Comment:
Rather than hard-coding the types (with field names), I was thinking this could be more dynamic.

Something like the following for a single field error:

``` ts
export const FieldErrorResultsType = new GraphQLObjectType<
  any,
  ResolverContext
>({
  name: "FieldErrorResults",
  fields: () => ({
    name: {
      type: new GraphQLNonNull(GraphQLString),
    },
    message: {
      type: new GraphQLNonNull(GraphQLString),
    },
  }),
})
```

Then the type for `fieldErrors` could be a list of these, and the resolver could iterate over the error/detail and create the desired array: `[{name: "email", message: "not a valid email"}, {name: "location", message: "not a valid location"}]`, etc.

Reply:
ie- the SDL would be:

```
type GravityMutationError {
  fieldErrors: [FieldErrors]
}

type FieldErrors {
  name: String!
  message: String!
}
```